### PR TITLE
adds a rather barebones timing api and some tests

### PR DESCRIPTION
**Pull request by noahadams**

### CODE REVIEW

**Comment by rrjamie** [comment]
this looks good to me. @jansepar can you take a quick look?

File: Gruntfile.js [hunk 1]
```diff
@@ -116,7 +136,8 @@ module.exports = function(grunt) {
                         'http://localhost:3000/tests/resizeImages.html',
                         'http://localhost:3000/tests/jazzcat.html',
                         'http://localhost:3000/tests/unblockify.html',
-                        'http://localhost:3000/tests/cssOptimize.html'
+                        'http://localhost:3000/tests/cssOptimize.html',
+                        'httpp://localhost:3000/tests/timing.js'
```
Comment (jansepar) [inline Gruntfile.js]: extra p

**Comment by jansepar** [comment]
While I do feel that creating a new library that includes the timing API is the wrong approach, I think it's fine for now. Eventually we should just be creating a custom mobify.js bundle for Swift that also requires in the swift processor stuff (which isn't that far away).

**Comment by noahadams** [comment]
@jansepar I basically agree, but this is the quick and dirty approach that will work most easily for us now with a  minimum of consequence.

**Comment by jansepar** [comment]
Yep agreed :) I'm going to make sure everything is still working and run the tests on Sauce before I give my +1

**Comment by jansepar** [comment]
:+1:

**Pull request merged**